test/e2e/autotls/auto_tls_test.go

```diff
@@ -115,13 +119,86 @@ func httpsReady(svc *servingv1.Service) (bool, error) {
 	}
 }
 
-func createRootCAs(t *testing.T, clients *test.Clients, ns, secretName string) *x509.CertPool {
+func TestRouteModified(t *testing.T) {
```

Comment:
Will this cause the test running under different environment has the same name? we probably want to follow the configuration in https://github.com/knative/serving/pull/6856/files#diff-462893639ceafde67ae16b6a81365509R61